src/test/groovy/io/codearte/jfairy/producer/company/locale/CompanyProviderSpec.groovy

```diff
@@ -0,0 +1,82 @@
+package io.codearte.jfairy.producer.company.locale
+
+import io.codearte.jfairy.data.DataMaster
+import io.codearte.jfairy.producer.BaseProducer
+import io.codearte.jfairy.producer.VATIdentificationNumberProvider
+import io.codearte.jfairy.producer.company.CompanyProvider
+import spock.lang.Specification
+
+import static io.codearte.jfairy.producer.company.CompanyProvider.*
+
+class CompanyProviderSpec extends Specification {
+
+    def baseProducer = Mock(BaseProducer)
```

Comment:
I know this is completely subjective, but we are not big fans of "def"; could you use actual types, i.e. `BaseProducer baseProducer = Mock(BaseProducer)` . I think I have missed it and approved it in previous PRs, but I'm going to switch it. For the new one, please change it.